packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -471,53 +479,56 @@ class _BottomNavigationTile extends StatelessWidget {
 
     return Expanded(
       flex: size,
-      child: Semantics(
-        container: true,
-        selected: selected,
-        child: Stack(
-          children: <Widget>[
-            InkResponse(
-              onTap: onTap,
-              child: Padding(
-                padding: EdgeInsets.only(top: topPadding, bottom: bottomPadding),
-                child: Column(
-                  crossAxisAlignment: CrossAxisAlignment.center,
-                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                  mainAxisSize: MainAxisSize.min,
-                  children: <Widget>[
-                    _TileIcon(
-                      colorTween: colorTween,
-                      animation: animation,
-                      iconSize: iconSize,
-                      selected: selected,
-                      item: item,
-                      selectedIconTheme: selectedIconTheme,
-                      unselectedIconTheme: unselectedIconTheme,
-                    ),
-                    _Label(
-                      colorTween: colorTween,
-                      animation: animation,
-                      item: item,
-                      selectedLabelStyle: selectedLabelStyle,
-                      unselectedLabelStyle: unselectedLabelStyle,
-                      showSelectedLabels: showSelectedLabels,
-                      showUnselectedLabels: showUnselectedLabels,
-                    ),
-                  ],
+      child: Container(
+        color: selected ? activeBackgroundColor : Colors.transparent,
```

Comment:
Above you used
```suggestion
        color: selected ? activeBackgroundColor : null,
```
for `CupertinoTabBar`. Is there a reason for the difference here in `BottomNavigationBar`?